Services/AccountsService.js

```diff
@@ -1,8 +1,16 @@
 const getAccounts = (db, callback) => {
-    var collection = db.collection('accounts')
+    let collection = db.collection('accounts')
     collection.find({}).toArray((err, docs) => {
         callback(docs)
     })
 }
 
-module.exports.getAccounts = getAccounts
\ No newline at end of file
+const createAccount = (db, newAccountInfo, cb) => {
```

Comment:
Pick one (cb or callback) for naming parameters, you have used both in idfferent places

Reply:
good note, done